commit bdb73e428f774d5754408c0936df428e229d57c6 (refs/changes/36/336/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-08-23T22:04:47+00:00 (3 years, 2 months ago)
    
    joyent/binder#2 need to use https:// for submodules
    Reviewed by: Cody Mello <cody.mello@joyent.com>

diff --git a/.gitmodules b/.gitmodules
index cc3ec61..6c41273 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,21 +1,21 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/manta-scripts"]
 	path = deps/manta-scripts
-	url = git@github.com:joyent/manta-scripts.git
+	url = https://github.com/joyent/manta-scripts.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
-	url = git@github.com:joyent/sdc-scripts.git
+	url = https://github.com/joyent/sdc-scripts.git
 [submodule "deps/restdown-brand-remora"]
 	path = deps/restdown-brand-remora
-	url = git@github.com:joyent/restdown-brand-remora.git
+	url = https://github.com/joyent/restdown-brand-remora.git
 [submodule "deps/zookeeper-common"]
 	path = deps/zookeeper-common
-	url = git@github.com:joyent/zookeeper-common.git
+	url = https://github.com/joyent/zookeeper-common.git
